app/src/main/java/com/lagradost/cloudstream3/ui/player/AbstractPlayerFragment.kt

```diff
@@ -622,9 +630,20 @@ abstract class AbstractPlayerFragment(
             PlayerResize.Fill -> AspectRatioFrameLayout.RESIZE_MODE_FILL
             PlayerResize.Fit -> AspectRatioFrameLayout.RESIZE_MODE_FIT
             PlayerResize.Zoom -> AspectRatioFrameLayout.RESIZE_MODE_ZOOM
+            PlayerResize.SixteenByNine -> 16.0f/9.0f
+            PlayerResize.NineBySixteen -> 9.0f/16.0f
+            PlayerResize.OneByOne -> 1.0f/1.0f
+            PlayerResize.IMAX -> 1.43f / 1.0f
+            PlayerResize.DolbyVision -> 2.39f / 1.0f
+            PlayerResize.UltraWide -> 21.0f / 9.0f
+            PlayerResize.Expanded -> 1.90f / 1.0f
```

Comment:
With this many options a popup might be preferable over pressing resize 9 times, but this is a great first framework!

Reply:
I have made changes for the pop up, Attaching the video in the PR for reference let me know if I have to make any changes or suggestions